api/src/controllers/collection-controller.js

```diff
@@ -106,4 +252,29 @@ exports.protectedDelete = async function (args, res, next) {
 
   queryActions.sendResponse(res, 200, {});
   next();
+}
+
+const checkRecordExistsInCollection = async function (records, collectionId, editing = false) {
+  const db = mongodb.connection.db(process.env.MONGODB_DATABASE || 'nrpti-dev');
+  const collectionDB = db.collection('nrpti');
+  let promises = [];
+  for (const record of records) {
+    // does this record exit in any other collection?
+    const collectionCount = await collectionDB.countDocuments({ _schemaName: RECORD_TYPE.CollectionBCMI._schemaName, records: { $elemMatch: { $eq: new ObjectID(record) } } });
```

Comment:
trying to update an newly created and edited collection (note my records had no attached documents) the api crashes due to TypeError: collectionDB.countDocuments is not a function

Reply:
c4001780a78be7845c544c94e828bec4fb8b1900